### PR TITLE
Fix compliance issues for iOS builds

### PR DESCRIPTION
**Pull request by yanavlasov**
Risk Level: Low
Testing: Unit Tests
Docs Changes: N/A
Release Notes: N/A
Platform Specific Features: N/A

### CODE REVIEW

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!

I don't think this needs a cross-company approval, and I'll let you decide.

**Review by RyanTheOptimist — APPROVED** [review]

**Pull request merged**
